src/models/steg/omDeg/medlemskap.ts

```diff
@@ -16,7 +16,11 @@ export interface IUtenlandsopphold {
   id: string;
   periode: IPeriode;
   land?: ISpørsmålFelt;
+  erEøsLand: boolean;
   begrunnelse: ITekstFelt;
+  personidentUtland?: ITekstFelt;
+  harPersonidentUtland?: boolean;
```

Comment:
Det er viktig at erEøsLand og harPersonIdentUtland både blir mellomlagret og lagret ved innsending for at gjenbruk skal fungere